.github/CODEOWNERS

```diff
@@ -190,6 +190,10 @@
 # Design
 **/*.scss  @elastic/kibana-design
 
+# Enterprise Search
+/x-pack/plugins/enterprise_search/ @elastic/app-search-frontend @elastic/workplace-search-frontend
```

Comment:
Would appreciate some thoughts here - is it OK (for now?) if all CODEOWNER alerts go to both Workplace Search and App Search, or just App Search for now?

Alternatively, should I more painstakingly and granularly split up our file structure? Right now we have a lot of nested `app_search` folders that will have corresponding `workplace_search` siblings - do we want to eventually only alert AS FEDs for AS code, and WS for WS, etc.?

Reply:
I could see the benefits for both. My personal opinion is to go with the current implementation but as soon as we start branching off and have code in `workplace_search` then we can diverge and go with the latter, more robust solution.